### PR TITLE
adds persistent images

### PR DESCRIPTION
**Pull request by murkymode**
Yeet!

### CODE REVIEW

File: client/src/Components/PostModal.jsx [hunk 1]
```diff
@@ -6,6 +7,8 @@ import React, { useReducer, useContext } from 'react';
 import { makeStyles } from '@material-ui/core/styles';
 import SendRoundedIcon from '@material-ui/icons/SendRounded';
 import axios from 'axios';
+import S3FileUpload from 'react-s3';
```
Comment (jordanabbasi) [inline client/src/Components/PostModal.jsx:10]: delete this line

File: package.json [hunk 2]
```diff
@@ -55,7 +55,10 @@
     "@material-ui/core": "^4.12.3",
     "@material-ui/icons": "^4.11.2",
     "@material-ui/lab": "^4.0.0-alpha.60",
+    "aws-sdk": "^2.982.0",
     "axios": "^0.21.1",
+    "browserify-fs": "^1.0.0",
```
Comment (jordanabbasi) [inline package.json:60]: delete

File: package.json [hunk 3]
```diff
@@ -55,7 +55,10 @@
...
+    "buffer": "^6.0.3",
```
Comment (jordanabbasi) [inline package.json:61]: delete

File: package.json [hunk 4]
```diff
@@ -65,10 +68,13 @@
     "morgan": "^1.10.0",
     "nodemailer": "^6.6.3",
     "nodemon": "^2.0.12",
+    "path-browserify": "^1.0.1",
```
Comment (jordanabbasi) [inline package.json:71]: delete

File: package.json [hunk 5]
```diff
@@ -65,10 +68,13 @@
...
     "pg": "^8.7.1",
     "react": "^17.0.2",
+    "react-aws-s3": "^1.5.0",
```
Comment (jordanabbasi) [inline package.json:74]: delete

File: package.json [hunk 6]
```diff
@@ -65,10 +68,13 @@
...
     "react-dom": "^17.0.2",
     "react-material-ui-carousel": "^2.3.1",
+    "react-s3": "^1.3.1",
```
Comment (jordanabbasi) [inline package.json:77]: delete

File: client/src/Components/PostModal.jsx (hunk 1 above)
Comment (murkymode) [reply client/src/Components/PostModal.jsx:10]: done

**Review by rgcuffee — APPROVED** [review]

**Review by jordanabbasi — APPROVED** [review]

**Pull request merged**